nuget/ValidationResult.cs

```diff
@@ -9,64 +11,49 @@ namespace glTF2Validator
     /// </summary>
     public sealed class ValidationReport
     {
-        internal ValidationReport(string srcPath, string rawReport)
+        public static ValidationReport Parse(string json)
         {
-            var lines = rawReport.Split('\n');
-
-            var status = 0;
-
-            var www = new List<string>();
-            var eee = new List<string>();
-
-            foreach (var l in lines)
+            var options = new JsonSerializerOptions
             {
-                if (l == "\tWarnings:") { status = 1; continue; }
-                if (l == "\tErrors:") { status = 2; continue; }
+                AllowTrailingCommas = true
```

Comment:
Why is this needed? If the report contains trailing commas, it's a bug.

Reply:
Ah, I'll remove it if it's not needed.